packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -118,7 +121,11 @@ function FontCollection( { slug } ) {
 
 	// NOTE: The height of the font library modal unavailable to use for rendering font family items is roughly 417px
 	// The height of each font family item is 61px.
-	const pageSize = Math.floor( ( window.innerHeight - 417 ) / 61 );
+	const windowHeight =
+		window.innerHeight <= MIN_WINDOW_HEIGHT
+			? MIN_WINDOW_HEIGHT
+			: window.innerHeight;
```

Comment:
```suggestion
	const windowHeight = Math.max( window.innerHeight, MIN_WINDOW_HEIGHT );
```

I think this is probably a matter of preference, so I think either is fine.